src/features/simd.md

```diff
@@ -0,0 +1,179 @@
+
+## WebAssembly SIMD Proposal
+
+The high level goal of the WebAssembly SIMD proposal is to introduce vector operations to the WebAssembly Specification, in a way that guarantees portable performance. 
+
+The set of SIMD instructions is large, and varied across architectures. The set of operations included in the WebAssembly SIMD proposal consist of operations that are well supported on a wide variety of platforms, and are proven to be performant. To this end, the current proposal is limited to standardizing Fixed-Width 128-bit SIMD operations. 
+
+The current proposal introduces a new v128 value type, and a number of new operations that operate on this type. The criteria used to determine these operations are -
+
+* The operations should be well supported across multiple modern architectures.
+
+* Performance wins should be positive across multiple relevant arches within an instruction group.
+
+* The chosen set of operations should minimize performance cliffs if any.
+
+ 
+
+The proposal is in active development, both V8, and the toolchain have working prototype implementations for experimentation. As these are prototype implementations, they are subject to change as new operations are added to the proposal.
+
+## Using WebAssembly SIMD
+
+### Enabling experimental SIMD support in Chrome
+
+WebAssembly SIMD support is prototyped behind a flag in chrome, to try out the SIMD support on the browser, pass --enable-features=WebAssemblySimd, or toggle the WebAssemblySimd flag in chrome://flags. This work is bleeding edge, and continuously being worked on. To minimize the chances of breakage, please use the latest version of the toolchain as detailed below, and a recent Chrome Canary. If something doesn’t look right, please file a bug at crbug.com/v8.
+
+### Building C / C++ to target SIMD
+
+WebAssembly’s SIMD support depends on using a recent build of clang with the WebAssembly LLVM backend enabled. Emscripten has support for the WebAssembly SIMD proposal as well. Install, and activate the latest-upstream distribution of emscripten using [emsdk](https://emscripten.org/docs/getting_started/downloads.html) to use the bleeding edge SIMD features.
```

Comment:
The whole text seems to use ’ instead of ' (some UTF-8 apostrophe instead of ascii). Not sure if this is intended.

Reply:
That's awesome, and what we do for all of v8.dev (at least everything that went through me) :)